input/pagecontent/transport.md

```diff
@@ -1 +1,56 @@
-here is a sample.
+## GenomeX data transport
+
+Precision Medicine workflows require sharing complex data types with many relationships.
+Multiple transmission formats and protocols are used to transport Genomic test results from laboratories to health care organizations.
+HL7v2 transmission, using the MLLP protocol, is ubiquitously adopted as a result of healthcare law and remains the only guaranteed transport 
+available to support data exchange for Clinical Workflows, especially Lab Orders & Results.
+
+FHIR GenomeX data structures will be transmitted to enable new precision medicine workflows. In the prescribed options EHR integrations w/ Labs 
+will continue to support HL7v2 Ordering along with new HTTP based transport options as listed below:
+
+1. HL7v2 Order & Result via embedded FHIR bundle as base 64 string
+2. HL7v2 Order & Result via FHIR APIs links: HL7 ORU contains FHIR server link to download the reports through HTTPs.
+3. HL7v2 Order & Result via FHIR API polling (batch download)
+
+### 1. HL7 ORU with embedded FHIR bundle as base 64 string
+
+Steps:
+* EHR system places a new order through ORM HL7 into lab  
+* Lab sends Observation (ORU) with a FHIR bundle as base 64 string embedded in
+OBX segment
+
+<object data="genomeX_transport_option_1.svg" type="image/svg+xml"></object>

```

Comment:
There is a space in actual file's filename. So the diagram does not show up. Here is good. Update the filename.